### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by yodatak** [semaphoreui/semaphore#1071] [Q&A]
I want to be able to take the inventory from the git repository , to have a single repository with my inventory and my playbooks is it something interesting to semaphore ?

**Comment by fiftin** [comment]
Hi @yodatak 
You can do this. Create/Open inventory, select Type **File** and specify path to you inventory file.

![image](https://user-images.githubusercontent.com/914224/198816092-e5830e5f-03b3-4a5e-a57c-62c549c18391.png)

**Reply by stdevel** [reply]
Hi @fiftin,
thanks for the feedback! Worked like charm for me.

Is there any chance to have this also reflected in the UI? I think a hint, that `File` doesn't need to be a local file - but also relative in the repository, would be very good for new users. Took me quite some time to figure it out and find this discussion.
